inst/mac_build.sh

```diff
@@ -3,8 +3,30 @@
 #do everything build_GRAN_all does for the mac build, then sync w/AWS
 #run this from the main package directory, not from ./inst!
 
-Rscript -e 'library(granbuild); build_GRAN_all(c('3.4', '3.3'))'
+pointer=/Library/Frameworks/R.framework/Versions/Current
+versions=(3.4 3.3)
+origVersion=$(readlink $pointer)
+for ver in ${versions[@]}
+do
+	ln -sfhv $ver $pointer #change version
 
+	#have to be a directory up - is there a better alternative here?
```

Comment:
Agreed that it would be very nice not to have to specify a path to the grantools package or its parent. The following seems to work for me, from within the grantools directory:
```
R CMD INSTALL --no-multiarch --with-keep.source ../grantools
```

Reply:
Interesting, that does seem to work.

Reply:
Oh I guess it just needs to be

Reply:
oh! yes, even better